0-common/igluctl/src/main/scala/com.snowplowanalytics.iglu/ctl/Command.scala

```diff
@@ -110,8 +110,9 @@ object Command {
   val lint = Opts.subcommand("lint", "Validate JSON schemas") {
     (input, skipWarnings, skipChecks).mapN(Lint.apply)
   }
+  val version = Opts.flag("version", "Display version").map(_ => VersionFlag())
```

Comment:
`Functor` defined `as` operation, so it could be `Opts.flag("version", "Display version").as(VersionFlag)`